docs/rest-api.yml

```diff
@@ -200,12 +200,12 @@ paths:
           type: string
       responses:
         '204':
-          description: The server has stopped
+          description: The user's notebook server has stopped
         '202':
-          description: The server has been asked to stop, but is taking a while
+          description: The user's notebook server has not stopped, but a stop request has been submitted
   /users/{name}/admin-access:
     post:
-      summary: Grant an admin access to this user's server
+      summary: Grant admin access to this user's notebook server
```

Comment:
This is specifically "an admin", as in "an administrator". It only gives one individual access to the server.